### ISSUE TITLE
Fatal error: Cannot redeclare at_remove_tax_parent_dropdown()

### ISSUE THREAD

**Issue by mariantanase** [WPGov/amministrazione-trasparente#7]
L'installazione in automatico della versione 8.0.10 non funziona.
Cercando di caricarla manualmente sopra quella esistente, sembra che lo installa ma Il plugin non può essere attivato perché genera un errore fatale.

`Fatal error: Cannot redeclare at_remove_tax_parent_dropdown() (previously declared in C:\wamp64\www\demo\wp-content\plugins\amministrazione-trasparente\amministrazionetrasparente.php:165) in C:\wamp64\www\demo\wp-content\plugins\amministrazione-trasparente-8.0.10\amministrazionetrasparente.php on line 180
Call Stack
	Time	Memory	Function	Location
1	0.0004	363712	{main}( )	...\plugins.php:0
2	2.8817	27523296	plugin_sandbox_scrape( $plugin = 'amministrazione-trasparente-8.0.10/amministrazionetrasparente.php' )	...\plugins.php:192`

PHP 8.1.20

**Comment by milesimarco** [comment]
Ciao @mariantanase,
dall'errore vedo che si crea una situazione in cui la nuova versione prende la cartella "amministrazione-trasparente-8.0.10" anziché "amministrazione-trasparente". Non essendo sovrascritto il vecchio restano 2 plugin identici attivi (che generano errore).

Proverei a rimuovere la cartella con il suffisso "-8.0.10". È stata creata manualmente? (normalmente non dovrebbe esserci)

A presto,
M.

**Comment by mariantanase** [comment]
No, è stata generata in automatico quando ho tentato di installare il plugin manualmente (caricarlo).
La cartella del plugin esistente si chiamava 'amministrazione-trasparente'.
Cancellando la cartella con il suffisso -8.1.10 ha ripreso a funzionare.
Adesso mi sorge una domanda: Cancellando la cartella con il suffisso della versione come faccio ad essere sicuro che il plugin è stato aggiornato ? E perchè non ha funzionato l'aggiornamento automatico?

**Comment by milesimarco** [comment]
Opterei per sovrascrivere la cartella con il nuovo plugin (estraendo lo zip della 8.0.10).

L'aggiornamento è stato lanciato direttamente dalla bacheca di WordPress?

**Comment by mariantanase** [comment]
> L'aggiornamento è stato lanciato direttamente dalla bacheca di WordPress?

Si
